From fae3e5c9635835d80e0813c9ca6b4b515bf114a1 Mon Sep 17 00:00:00 2001
From: Gzxhwq <gzxhwq@gmail.com>
Date: Wed, 23 Sep 2020 12:56:50 +0000
Subject: [PATCH 2/3] v2ray: Fix 4.28.2 PKG_HASH

---
 package/lean/v2ray/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/lean/v2ray/Makefile b/package/lean/v2ray/Makefile
index 31b3383d3..429269560 100644
--- a/package/lean/v2ray/Makefile
+++ b/package/lean/v2ray/Makefile
@@ -15,7 +15,7 @@ PKG_BUILD_DIR:=$(BUILD_DIR)/v2ray-core-$(PKG_VERSION)
 
 PKG_SOURCE:=v2ray-core-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/v2fly/v2ray-core/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=e5d6603454622e1f466c4c20061992046124fc7c
+PKG_HASH:=e78446333fd28ef8a3ec2c25612458bfc024a226008888ed5277201136f4ca97
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
-- 
2.20.1

