From 684117e19231cfbaf3ad8661b0c7f2bbb6361fc6 Mon Sep 17 00:00:00 2001
From: Beginner <70857188+Beginner-Go@users.noreply.github.com>
Date: Fri, 6 Nov 2020 22:04:49 +0800
Subject: [PATCH] v2ray : update v4.32.1

---
 package/lean/v2ray/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/package/lean/v2ray/Makefile b/package/lean/v2ray/Makefile
index fb1364b61f7..fadf7870cc6 100644
--- a/package/lean/v2ray/Makefile
+++ b/package/lean/v2ray/Makefile
@@ -9,13 +9,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=v2ray
-PKG_VERSION:=4.32.0
+PKG_VERSION:=4.32.1
 PKG_RELEASE:=1
 PKG_BUILD_DIR:=$(BUILD_DIR)/v2ray-core-$(PKG_VERSION)
 
 PKG_SOURCE:=v2ray-core-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/v2fly/v2ray-core/tar.gz/v$(PKG_VERSION)?
-PKG_HASH:=a3d04aade039a3945e3704e674d61433e125e4b54e32cca9e794915aff3ca4c4
+PKG_HASH:=f5553ffd04ff226573e8132d9beaa63f4d8f4882eba047225b33620848bc6917
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=LICENSE
