diff --git a/feeds/luci/applications/luci-app-turboacc/root/etc/init.d/turboacc b/feeds/luci/applications/luci-app-turboacc/root/etc/init.d/turboacc
index e5f1454..8daa7e8 100755
--- a/feeds/luci/applications/luci-app-turboacc/root/etc/init.d/turboacc
+++ b/feeds/luci/applications/luci-app-turboacc/root/etc/init.d/turboacc
@@ -293,11 +293,11 @@ start(){
 			start_dnsproxy
 		fi
 
-		change_dns
+		# change_dns
 
 		/usr/share/dnscache/dnscache-while.sh > "/var/log/dnscache.file" 2>&1 &
 	else
-		revert_dns
+		# revert_dns
 		stop_dnscache
 	fi
 
@@ -323,7 +323,7 @@ stop(){
 	unload_sfe
 
 	stop_dnscache
-	revert_dns
+	# revert_dns
 
 	if [ "${restart_utils}" = "true" ]; then
 		echo "DNSMASQ revert"
