diff --git a/target/linux/generic/config-6.0 b/target/linux/generic/config-6.0
index e812b6a0d..98c083415 100644
--- a/target/linux/generic/config-6.0
+++ b/target/linux/generic/config-6.0
@@ -3226,6 +3226,9 @@ CONFIG_LOG_CPU_MAX_BUF_SHIFT=12
 # CONFIG_LPC_ICH is not set
 # CONFIG_LPC_SCH is not set
 # CONFIG_LP_CONSOLE is not set
+CONFIG_LRU_GEN=y
+CONFIG_LRU_GEN_ENABLED=y
+# CONFIG_LRU_GEN_STATS is not set
 # CONFIG_LSI_ET1011C_PHY is not set
 CONFIG_LSM="lockdown,yama,loadpin,safesetid,integrity"
 CONFIG_LSM_MMAP_MIN_ADDR=65536
